src/main/java/AddCommand.java

```diff
@@ -0,0 +1,79 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents the Command for adding Tasks and its subclasses.
+ * A subclass of Command.
+ */
+public class AddCommand extends Command {
+
+    private String taskCmd;
+    final String timePattern = "d MMMM yyyy, h:mma";
+    DateTimeFormatter dateTimeFormat = DateTimeFormatter.ofPattern(timePattern);
```

Comment:
Good use of the java DateTimeFormatter! However, I think that you could try implementing your own DateTimeFormatter class too!